### PR TITLE
updated segmentation under juliaImages column

### LINKED ISSUE #22: Comparision table in the docs needs to be updated
issuejulia2

There had been considerable works on image segmentation algorithms in JuliaImages. Though the segmentation section under "JuliaImages" column is still not updated accordingly.

### PR DESCRIPTION
**Pull request by tosalonijain**
Fixes #22 
I have tried to make suitable updates in the table in the segmentation block by adding the associated functions now available in juliaImages under the respective column. Looking forward for any changes to be addressed. 
Apologies for the passivity in my work since the introduction of the issue by me. 
Waiting for a response from the other side.

### CODE REVIEW

**Comment by waldyrious** [comment]
You seem to have included a backup file (`docs/src/api_comparison.md~`) in the commit.

File: docs/src/api_comparison.md [hunk 1]
```diff
@@ -88,9 +88,9 @@ welcome.
 | **Segmentation**                      |                                           |                                   |                                           |
 | Connected components                  | `label_components`                        | `label`                           | `bwconncomp`, `bwlabel`                   |
 | Foreground/background                 |                                           | `active_contour`                  | `activecontour`                           |
-| Clustering                            |                                           | `quickshift`                      |                                           |
+| Clustering                            | ` kmeans`, `fuzzy_cmeans`                 | `quickshift`                      |                                           |
```
Comment (timholy) [inline docs/src/api_comparison.md]: Maybe we should add `mean_shift` to the Julia column, `slic` to the python column.

File: docs/src/api_comparison.md [hunk 2]
```diff
@@ -88,9 +88,9 @@ welcome.
...
 | Marker segmentation                   |                                           | `random_walker`                   | `imsegfmm`                                |
```
Comment (timholy) [inline docs/src/api_comparison.md]: Perhaps add `seeded_region_growing` to the Julia column?

File: docs/src/api_comparison.md (hunk 1 above)
Comment (tosalonijain) [reply docs/src/api_comparison.md]: @timholy  under Clustering section of segmentation or else where??

File: docs/src/api_comparison.md (hunk 2 above)
Comment (tosalonijain) [reply docs/src/api_comparison.md]: Same doubt as above.

File: docs/src/api_comparison.md (hunk 1 above)
Comment (timholy) [reply docs/src/api_comparison.md]: Yeah, that's a clustering-based algorithm.

File: docs/src/api_comparison.md (hunk 2 above)
Comment (timholy) [reply docs/src/api_comparison.md]: To `Marker segmentation`

**Comment by timholy** [comment]
Thanks! The only remaining issue is to `git rm api_comparison.md~` (the  backup file) as @waldyrious noted. Then I can merge this.

**Comment by tosalonijain** [comment]
@timholy I have removed the backup file as suggested by @waldyrious . 
I hope it seems okay now to be merged.
Thanks!

**Comment by timholy** [comment]
Great, many thanks for tackling this!

**Pull request merged**